docs/source/_comparison_generator.py

```diff
@@ -108,6 +108,13 @@ def _section(
 _dtype_na = (
     '`object` and string dtypes are not supported in GPU and thus'
     ' left unimplemented in CuPy.')
+_dtype_time = (
+    '`datetime64` and `timedelta64` dtypes are currently unsupported.')
+_dtype_rec = (
+    'Structured arrays and record arrays are currently unsupported.')
+_float_error_state = (
+    'Floating point error handling depends on CPU-specific features'
+    ' which is not available in GPU.')
```

Comment:
By the way, `raise NotImplementedError` in `cupyx.seterr` (in `cupyx/_ufunc_config.py`) could have the same message.